### PR TITLE
Do kommander catlog api bumps to kommander chart

### PR DESCRIPTION
**Pull request by faiq**
**What type of PR is this?**

CI Automation Chore

**What this PR does/ why we need it**:

Auto syncs kommander catalog api in kommander chart's requirements 

**Which issue(s) this PR fixes**:

https://jira.d2iq.com/browse/D2IQ-71918

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Checklist**

* [x] *If a chart is changed, the chart version is correctly incremented.*
* [x] The commit message explains the changes and why are needed.
* [x] The code builds and passes lint/style checks locally.
* [x] The relevant subset of integration tests pass locally.
* [x] The core changes are covered by tests.
* [x] The documentation is updated where needed.

### CODE REVIEW

**Review by joejulian — APPROVED** [review]

File: Dispatchfile [hunk 1]
```diff
@@ -157,7 +157,17 @@ git_resource(
     url="https://github.com/mesosphere/" + kubeaddons_kommander_git,
     revision="master")
 do_bump_kommander = bump_addon("bump-kommander", kubeaddons_kommander_git)
-
+do_bump_catalog="bump-kommander-catalog-api"
+do_bump_kommander_catalog_api = bump_requirements(do_bump_catalog, git, "stable/kommander",
+  command=["/bin/sh", "-c"], args=[
+    """
+  git config --local user.name 'Dispatch CI'
```
Comment (cprovencher) [inline Dispatchfile:164]: missing GPG stuff to sign the commits

**Review by cprovencher — CHANGES_REQUESTED** [review]

File: Dispatchfile [hunk 2]
```diff
@@ -157,7 +157,17 @@ git_resource(
     url="https://github.com/mesosphere/" + kubeaddons_kommander_git,
     revision="master")
 do_bump_kommander = bump_addon("bump-kommander", kubeaddons_kommander_git)
-
+do_bump_catalog="bump-kommander-catalog-api"
+do_bump_kommander_catalog_api = bump_requirements(do_bump_catalog, git, "stable/kommander",
+  command=["/bin/sh", "-c"], args=[
```
Comment (cprovencher) [inline Dispatchfile:162]: I like the idea of passing the args here

**Review by cprovencher — APPROVED** [review]

**Pull request merged**